hvantk/tools/enrichex/burden_cli.py

```
@@ -27,6 +27,112 @@ def register_burden_commands(group):
+    sets and renders an ASCII table of the top hits, with columns chosen by
+    phenotype type (odds ratio for binary, standard error for continuous).
+    """
+    n_significant = result_df["significant"].sum()
+    click.echo("\n" + "=" * 60)
+    click.echo("BURDEN ANALYSIS SUMMARY")
+    click.echo("=" * 60)
+    click.echo(f"Total gene sets tested: {len(result_df)}")
+    click.echo(f"Significant gene sets (p_adj < {alpha}): {n_significant}")
+
+    if n_significant > 0:
+        click.echo(f"\nTop significant gene sets:")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)